pkg/githubapp/githubapp.go

```diff
@@ -48,6 +50,10 @@ func (g *GitHubApp) PersistentFlags(p *pflag.FlagSet) {
 		"Callback webserver listen address")
 	p.IntVar(&g.webServerPort, "webserver-port", g.webServerPort,
 		"Callback webserver port number")
+
+	if err := cmd.MarkPersistentFlagRequired("org"); err != nil {
```

Comment:
You need to clean up the `Validate()` method, as the check for `gitHubOrgName` becomes redundant.